more_autograding_examples/python_custom_validation/config/custom_validation_code/grader.py

```diff
@@ -1,129 +1,250 @@
+"""
```

Comment:
I think this should all go on submitty.org instead?

Reply:
Moved relevant potion to Submitty.org (PR #124).